index.js

```diff
@@ -92,6 +93,15 @@ Parser.prototype._transform = function(file, encoding, done) {
 
 
 
+    // we find the relative path
+    // =========================
+    var relativeFilePath = path.relative(__dirname, this.base);
+    if (file.path) {
```

Comment:
In many of the tests, we're setting the 'base' of the fakeFiles we use, but not the path. This is why I initialized with this.base and conditionally override after that. So not a use case per say, but for the test cases, yes.